ci/get_features.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
This is added as `python` in the first commit, and then fixed to `python3` in the last commit. Better to just make it correct when the script is added.

Reply:
Oops, completely issed this, will fix.

Reply:
Ah, squashed into the wrong commit! :-P